source/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(ZividCppSamples)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 include(CompilerOptions)
 
-set(ZIVID_VERSION 2.2.0)
+set(ZIVID_VERSION 2.3.0)
```

Comment:
Is this in conflict with our statement: Note: The samples require Zivid SDK v2 (minor version 2.2 or newer)?
I guess, we need to add stg like. To use version x.y.z change CMakeLists to x.y.z?

Reply:
I checked and this check in CMake is not EXACT. Not entirely sure what it means (although it's explained here: https://cmake.org/cmake/help/latest/command/find_package.html#version-selection). In any case. I tested, and it works with both 2.2 and 2.3 if we leave it at 2.2. If we update to 2.3 then it only works with 2.3. It looks like it checks for minimum compatibility.

Reply:
That is great.